source/_integrations/gardena.markdown

```diff
@@ -0,0 +1,78 @@
+---
+title: "Gardena"
+description: "Instructions on how to setup Gardena devices within Home Assistant."
+logo: gardena.jpg
+ha_category:
+  - Hub
+  - Vacuum
+  - Sensor
+  - Switch
+ha_release: XXX
+ha_qa_scale: XXX
```

Comment:
Please check the quality scale index for the level it supports and update the documentation accordingly. (or remove in case it doesn't match any of the levels).

Reply:
Ok, I will put silver, but the component is fully async ... so not sure what to do with that. Silver will be fine until it can be configured via the UI (i have never done it).